APIs/schemas/constraint_set.json

```diff
@@ -20,6 +20,13 @@
       "description": "This value indicates whether a Constraint Set is available to use immediately (true) or whether this is an offline capability which can be activated via some unspecified configuration mechanism (false). When the attribute is omitted its value is assumed to be true.",
       "type": "boolean",
       "default": true
+    },
+    "urn:x-nmos:substreams": {
```

Comment:
We've discussed that in order to maintain BCP-004-01 v1.0 compatibility this cannot be `urn:x-nmos:cap:substreams` because of the _pattern property_ below. But not clear whether `urn:x-nmos:substreams` is precise enough.